### DISCUSSION TITLE
Reflect fields types from schema graphql

### DISCUSSION THREAD

**Discussion by LucianViana** [graphql-java/graphql-java#3638] [General]
There is already some initiative in graphql-java to generate tables in the database based on the graphql schema types fields using reflect as in the video?
Dynamic GraphQL with Go - Kenneth Shaw
https://youtu.be/GEaU-0J_2m0?si=Y5lIRyMMFexYu9FD
Thanks

**Comment by bbakerman** [comment]
Is this question meant to be

> Is there already some initiative in graphql-java to generate tables in the database based on the graphql schema types fields 

if so, the answer is no and will not be.  The graphql-java library is mostly an engine - its not a framework like Spring Graphql and DGS et al.

We would not do database table generation in the graphql-java engine library.

That said for some this might not be a crazy idea but thats for some other framerwork type code to do not us.
